v2/components/schemas/QuotaGroup.yaml

```diff
@@ -0,0 +1,60 @@
+# SPDX-License-Identifier: GPL-3.0
+# SPDX-FileCopyrightText: Canonical Ltd
+
+type: object
+description: Defines a resource quota group for one or more snaps.
+properties:
+  group-name:
```

Comment:
group name is required, as in always expected in requests, but also always included in responses,, so this could be listed in top level:
```
required:
  - group-name
```